pkg/azure/core_test.go

```diff
@@ -1683,7 +1673,7 @@ func UnmarshalInterfacesDeleteFuture(bytesUnterfacesFutureAPI []byte) network.In
 
 func newMachine(name string) *v1alpha1.Machine {
 	return &v1alpha1.Machine{
-		ObjectMeta: metav1.ObjectMeta{
+		ObjectMeta: v1.ObjectMeta{
```

Comment:
You could revert this change. The idiomatic import alias for "k8s.io/apimachinery/pkg/apis/meta/v1" is `metav1`, not `v1`.